data/ecosystems/a/avalanche.toml

```diff
@@ -143,6 +143,7 @@ sub_ecosystems = [
   "Venly",
   "Verso Finance",
   "Wonderland",
+  "WOOFi",
```

Comment:
doesn't look like we have an ecosystem called `WOOFi`, can you remove this line or create an ecosystem with this exact name and the corresponding repos?

Reply:
WOOFi's repos are under WOO Network which is the project name. The repos I linked are specific to WOOFi the DeFi product of WOO Network

Reply:
what I mean is that a toml file must exist with title WOOFi for you to be able to add this line. Since it doesn't exist the checks in the PR are failing, you will need to create a .toml file for such subecosystem to be able to add it here. Until then, the PR checks will fail and we can't merge

Reply:
i updated the woo-network.toml to woofi.toml so it has the correct name. does that work

Reply:
You'd have to create a new toml file for woofi, add it as a sub-ecosystem to woo-network and then move the repos that are currently under woo-network to woofi when it proceeds.